web_server.py

```diff
@@ -0,0 +1,93 @@
+
+"""
+This benchmark runs a trivial Twisted Web server and client and makes as many
+requests as it can in a fixed period of time.
+
+A significant problem with this benchmark is the lack of persistent connections
+in the HTTP client.  Lots of TCP connections means lots of overhead in the
+kernel that's not really what we're trying to benchmark.  Plus lots of sockets
```

Comment:
If you want to make these into high quality benchmarks, fixing the persistent http connection shortcoming would go a long way.